docs/essentials/parameterized-promotions.md

```diff
@@ -0,0 +1,193 @@
+---
+Description: Use parameterized promotions to improve code reusability and to dynamically configure your promotions and deployments.
+---
+
+# Parameterized Promotions
+
+The **parameterized promotions** feature allows users to pass parameters to promotion
+pipelines. You can do this either via a form in the UI, 
+via the Semaphore API, or on auto-promotions by setting up default values.
+
+The core advantage of this feature is **code reusability**. It is no longer necessary to create many promotion pipelines with almost identical configurations. Users can now create a single pipeline and rely on parameters for smaller but critical differences.
+
+## Configuring the promotion parameters
+Let's start with defining the parameters that our promotion will support. 
+
+In your `.yml` file, when defining the promotion you can set up
+as many parameters as you want. 
+
+Possible options include:
+
+- **required**: values (true/false) - If the parameter is required and
+default_value is not set the promotion won’t start.
+- **default_value** - Can only be set if the parameter is required, it
+is a default value of the parameter.
+- **options** - Use this to define a list of all possible values and the user will have to select one of them from the dropdown menu. If omitted, the user will be able to input any value via the input box.
```

Comment:
Use this to define a list of all possible values. The user needs to select a value from the dropdown menu. If omitted, the user is able to input any value via the input box.

changed to present tense and used shorter sentences.